packages/@okta/i18n/src/properties/login.properties

```diff
@@ -616,6 +616,9 @@ deviceTrust.sso.expire.button = Sign in using Okta Mobile
 deviceTrust.universalLink.fallback.getOktaMobile.title = Get Okta Mobile
 deviceTrust.universalLink.fallback.getOktaMobile.subtitle = Go to the {0}AppStore{1}, {0}search{1} for {0}Okta Mobile{1} and tap on {0}GET{1} Okta Mobile. Once installed, sign in to Okta Mobile and follow the instructions to secure your device.
 
+# Device Challenge Probing and Polling
+deviceChallengePoll.title = Sign in
```

Comment:
We'hv already got a few tokens in this file for `Sign in`
https://github.com/okta/okta-signin-widget/blob/a40c24fb3d1897d6537a4b0c31b9468bb413a9be/packages/%40okta/i18n/src/properties/login.properties#L477
https://github.com/okta/okta-signin-widget/blob/a40c24fb3d1897d6537a4b0c31b9468bb413a9be/packages/%40okta/i18n/src/properties/login.properties#L482
and this seems like another new key with the same value. Can we update this to be something more generic like `signin.title = Sign in`  so that we can potentially re-use this in the future or re-use the existing one ?